### PR TITLE
ES - Translated EN-ES

### PR DESCRIPTION
**Pull request by MarianCP**
Translated strings 175, 441, 469, 488, 498, 514, 516, 517

### CODE REVIEW

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 1]
```diff
@@ -172,7 +172,7 @@
 <!ENTITY fontCourier.label "Courier">
 <!ENTITY fontCourier.accesskey "C">
 <!ENTITY FontSizeSelect.tooltip "Elija un tamaño de letra">
-<!ENTITY resetToDefaultFontSize.label "Reset to default">
+<!ENTITY resetToDefaultFontSize.label "Volver a la configuración predeterminada">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 2]
```diff
@@ -438,7 +438,7 @@
 -->
 
 <!ENTITY listprops.accesskey "l">
-<!ENTITY tableFix.label "Fix Table Layout">
+<!ENTITY tableFix.label "Fijar diseño de la tabla">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: POSSIBLE CORRECTION: This has been translated as "Set table layout" . Does Fix here refer to the action of fixing the layout?

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 3]
```diff
@@ -466,9 +466,9 @@
 <!ENTITY clear.accesskey "l">
 <!ENTITY fileopen.accesskey "o">
 <!ENTITY formatlistmenu.accesskey "l">
-<!ENTITY insertIsIndexCmd.label "Index Search">
+<!ENTITY insertIsIndexCmd.label "Búsqueda del índice">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 4]
```diff
@@ -485,7 +485,7 @@
 <!ENTITY formatToolbar.italicChar "I">
 <!ENTITY decreaseindent.accesskey "D">
 <!ENTITY heading1.accesskey "1">
-<!ENTITY listProps.label "List Properties…">
+<!ENTITY listProps.label "Listar propiedades…">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 5]
```diff
@@ -495,7 +495,7 @@
 <!ENTITY formatfontfixed.keybinding "t">
 <!ENTITY tablemenu.accesskey "b">
 <!ENTITY insertlink.accesskey "l">
-<!ENTITY printPreviewCmd.label "Print Preview">
+<!ENTITY printPreviewCmd.label "Vista previa">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 6]
```diff
@@ -511,10 +511,10 @@
 <!ENTITY inserthline.accesskey "o">
 <!ENTITY decreaseindentkb "-">
 <!ENTITY insertinputimage.accesskey "I">
-<!ENTITY pasteTextCmd.label "Text">
+<!ENTITY pasteTextCmd.label "Texto">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 7]
```diff
@@ -511,10 +511,10 @@
...
 <!ENTITY newBlankPage.accesskey "P" >
-<!ENTITY textColorCaption.label "Text Color">
-<!ENTITY insertFormMenu.label "Form">
+<!ENTITY textColorCaption.label "Color del texto">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd [hunk 8]
```diff
@@ -511,10 +511,10 @@
...
+<!ENTITY insertFormMenu.label "Formulario">
```
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

File: es-ES/editor/ui/chrome/composer/editorOverlay.dtd (hunk 4 above)
Comment (CameliaF) [inline es-ES/editor/ui/chrome/composer/editorOverlay.dtd]: APPROVED

**Comment by shermandickman** [comment]
@mkaply This is ready to merge if we can resolve the conflicts.

**Pull request closed without merging**